### ISSUE TITLE
Typo in the example 14 HTML

### ISSUE THREAD

**Issue by GeneTso** [seiyria/bootstrap-slider#792]
In the example 14 http://seiyria.com/bootstrap-slider/#example-14

`<input id="ex14" type="text" data-slider-ticks="[0, 100, 200, 300, 400]" data-slider-ticks-snap-bounds="30" data-slider-ticks-labels='["$0", "$100", "$200", "$300", "$400"]' ticks_positions="[0, 30, 60, 70, 90, 100]" />`

`ticks_positions` should be `data-slider-ticks-positions`

**Comment by seiyria** [comment]
Thanks

**Comment by rovolution** [comment]
@GeneTso we would love a PR that addresses this!

**Comment by jespirit** [comment]
@rovolution If I wanted to edit the API examples HTML page, do I need to submit a pull request on the `gh-pages` branch?

**Comment by rovolution** [comment]
@jespirit nope. need to make the change in the `[index.tpl](https://github.com/seiyria/bootstrap-slider/blob/master/tpl/index.tpl)` file in the `master` branch and open a PR against our `master` branch, which is the template file that the `gh-pages` page is generated from.
